src/Network/Network/Models/NetworkManager/PSNetworkManagerSecurityConfiguration.cs

```diff
@@ -23,6 +23,8 @@ public class PSNetworkManagerSecurityConfiguration : PSNetworkManagerBaseResourc
     {
         public string SecurityType { get; set; }
 
+        public List<string> NetworkIntentPolicyBasedService { get; set; }
+
         public string DeleteExistingNSGs { get; set; }
     }
 }
```

Comment:
Since in swagger, we have split it to adminconfig and userconfig, it would be better to split model to two here also.,

Reply:
was not split in this API version, it will be split in 2022-02-01-preview